calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -215,6 +213,9 @@ protected function generateFiles($arrFeed)
 			$request->attributes->set('_scope', 'frontend');
 		}
 
+		global $objPage;
+		$origObjPage = $objPage;
```

Comment:
Would `$objOrigPage` be more consistent with the old naming schema?
```suggestion
		$objOrigPage = $objPage;
```

Reply:
I consider `$objPage` a special variable name in this context and thus find it more consistent to say "the original global objPage value". And it would be consistent with `$origScope` as well (though that variable lacks the type prefix of course).